package.json

```
@@ -29,18 +29,23 @@
         "build:stub": "cross-env ENVIRONMENT=prod BUILD=stub rollup --config rollup.config.js",
         "build:snippet": "uglifyjs snippet.js -nm -o snippet.min.js",
         "build:test-bundle": "cross-env TESTTYPE=main ENVIRONMENT=prod rollup --config rollup.test.config.js",
-        "build:browserify:cjs": "browserify test/integrations/cjs/browserify/index.js -o test/integrations/cjs/dist/browserify-output.js  && npm run test:karma:browserify:cjs",
-        "build:rollup:cjs": "rollup --config test/integrations/cjs/rollup/rollup.config.js  && npm run test:karma:rollup:cjs",
-        "build:webpack:cjs": "webpack --config test/integrations/cjs/webpack/webpack.config.js && npm run test:karma:webpack:cjs",
-        "build:rollup:module": "rollup --config test/integrations/module/rollup/rollup.config.js  && npm run test:karma:rollup:module",
-        "build:webpack:module": "webpack --config test/integrations/module/webpack/webpack.config.js && npm run test:karma:webpack:module",
+        "build:browserify:cjs": "browserify test/integrations/cjs/browserify/index.js -o test/integrations/cjs/dist/browserify-output.js",
+        "build:rollup:cjs": "rollup --config test/integrations/cjs/rollup/rollup.config.js",
+        "build:webpack:cjs": "webpack --config test/integrations/cjs/webpack/webpack.config.js",
+        "build:rollup:module": "rollup --config test/integrations/module/rollup/rollup.config.js",
+        "build:webpack:module": "webpack --config test/integrations/module/webpack/webpack.config.js",
         "build:ts": "tsc -p .",
         "test": "npm run build && npm run build:test-bundle && cross-env DEBUG=false karma start test/karma.config.js",
         "test:debug": "cross-env DEBUG=true karma start test/karma.config.js",
         "test:stub": "cross-env TESTTYPE=stub ENVIRONMENT=prod rollup --config rollup.test.config.js && karma start test/stub/karma.stub.config.js",
         "test:integrations": "npm run test:requirejs && npm run test:integrations:cjs && npm run test:integrations:module",
-        "test:integrations:cjs": "npm run build:browserify:cjs && npm run build:webpack:cjs && npm run build:rollup:cjs",
+        "test:integrations:cjs": "npm run test:integrations:cjs:browserfy && npm run test:integrations:cjs:webpack && npm run test:integrations:cjs:rollup",
+        "test:integrations:cjs:browserfy": "npm run build:browserify:cjs && npm run test:karma:browserify:cjs",
+        "test:integrations:cjs:webpack": "npm run build:webpack:cjs && npm run test:karma:webpack:cjs",
+        "test:integrations:cjs:rollup": "npm run build:rollup:cjs && npm run test:karma:rollup:cjs",
         "test:integrations:module": "npm run build:webpack:module && npm run build:rollup:module",
```

Without updating this to the new `test:integrations:module:*`, it will only build the files and note test given you updated the `build` commands:
```suggestion
        "test:integrations:module": "npm run test:integrations:module:webpack && npm run test:integrations:module:rollup",
```